license-tool.toml

```diff
@@ -0,0 +1,21 @@
+[overrides]
+"backon" = { origin = "https://github.com/Xuanwo/backon" }
+"bollard-stubs" = { origin = "https://github.com/fussybeaver/bollard" }
+"openssl-macros" = { origin = "https://github.com/sfackler/rust-openssl" }
+"serde_nanos" = { origin = "https://github.com/caspervonb/serde_nanos" }
```

Comment:
Why are these needed?

Reply:
The sources don't actually include either a `repository` or `homepage` line in their respective metadata, so we can't automatically determine where they are actually hosted.

Reply:
Worth PRing these few upstream to reduce the manual work here?

Reply:
Yeah, probably.